scripts/language_model/language_model.rst

```diff
@@ -156,20 +156,35 @@ The key features used to reproduce the results for pre-trained models are listed
 
 The dataset used for training the models is Google's 1 billion words dataset.
 
++-----------------+---------------+
 | Model           | LSTM-2048-512 |
-| --------------- | ------------- |
++=================+===============+
+| Mode            | LSTM          |
```

Comment:
The cell used is LSTMP, which is different from LSTM cells. Would make the mode "LSTMP" less confusing?